eZ/Publish/Core/MVC/Symfony/Security/Authentication/RepositoryAuthenticationProvider.php

```diff
@@ -38,7 +38,7 @@ public function setUserService(UserService $userService)
     protected function checkAuthentication(UserInterface $user, UsernamePasswordToken $token)
     {
         if (!$user instanceof EzUserInterface) {
-            return parent::checkAuthentication($user, $token);
+            parent::checkAuthentication($user, $token);
```

Comment:
That would be incorrect, as I assume this is for early return, since this class is unable to handle users of types other than `EzUserInterface`.
```suggestion
            parent::checkAuthentication($user, $token);
            
            return;
```